packages/app/ui/buttons/RadioButton.tsx

```diff
@@ -40,24 +40,23 @@ export function RadioButton({
   value,
   checked,
   name,
-  onChange,
+  onChange
 }: RadioButtonProps) {
   return (
-    <div className="flex">
+    <div className={twMerge("flex", className)}>
```

Comment:
With Diogo comment we don't need this className, right? Also we should pass the className to the button style and not to the parent component.

Reply:
Yes, it's something I forgot to revert, good catch, thanks!